### PR TITLE
View and router frontend CS

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issues at: https://travis-ci.org/yvesh/weblinks/jobs/140624371

#### Summary of Changes

CS

#### Testing Instructions

please review the code as it just CS fixes. If you want you can test the frontend of com_weblinks

### CODE REVIEW

File: src/components/com_weblinks/router.php [hunk 1]
```diff
@@ -259,7 +259,11 @@ public function parse(&$segments)
 /**
  * Weblinks router functions
  *
- * These functions are proxys for the new router interface
+ * @param   array  &$query  An array of URL arguments
+ *
+ * @return  array  The URL arguments to use to assemble the subsequent URL.
+ *
+ * Note. These functions are proxys for the new router interface
```
Comment (chrisdavenport) [inline src/components/com_weblinks/router.php]: "proxys" should be "proxies".

File: src/components/com_weblinks/router.php [hunk 2]
```diff
@@ -271,6 +275,18 @@ function WeblinksBuildRoute(&$query)
 	return $router->build($query);
 }
 
+/**
+ * Weblinks router functions
+ *
+ * @param   array  &$segments  The segments of the URL to parse.
```
Comment (chrisdavenport) [inline src/components/com_weblinks/router.php]: "&$segments" should be "$segments".

File: src/components/com_weblinks/router.php [hunk 3]
```diff
@@ -271,6 +275,18 @@ function WeblinksBuildRoute(&$query)
...
+ *
+ * @return  array  The URL attributes to be used by the application.
+ *
+ * Note. These functions are proxys for the new router interface
```
Comment (chrisdavenport) [inline src/components/com_weblinks/router.php]: "proxys" should be "proxies".

File: src/components/com_weblinks/views/category/tmpl/default_items.php [hunk 4]
```diff
@@ -84,87 +83,80 @@
...
 						<?php if (($this->params->get('show_link_description')) and ($item->description != '')) : ?>
 						<?php $images = json_decode($item->images); ?>
 						<?php  if (isset($images->image_first) and !empty($images->image_first)) : ?>
 						<?php $imgfloat = (empty($images->float_first)) ? $this->params->get('float_first') : $images->float_first; ?>
 						<div class="img-intro-<?php echo htmlspecialchars($imgfloat); ?>"> <img
-							<?php if ($images->image_first_caption):
-								echo 'class="caption"'.' title="' .htmlspecialchars($images->image_first_caption) .'"';
-							endif; ?>
+							<?php if ($images->image_first_caption) : ?>
+								<?php echo 'class="caption"' . ' title="' . htmlspecialchars($images->image_first_caption) . '"'; ?>
```
Comment (chrisdavenport) [inline src/components/com_weblinks/views/category/tmpl/default_items.php]: I think you can change
`'class="caption"' . ' title="'`
to just
`'class="caption" title="'`

File: src/components/com_weblinks/views/category/tmpl/default_items.php [hunk 5]
```diff
@@ -84,87 +83,80 @@
...
 							src="<?php echo htmlspecialchars($images->image_first); ?>" alt="<?php echo htmlspecialchars($images->image_first_alt); ?>"/> </div>
 						<?php endif; ?>
 						<?php  if (isset($images->image_second) and !empty($images->image_second)) : ?>
 						<?php $imgfloat = (empty($images->float_second)) ? $this->params->get('float_second') : $images->float_second; ?>
 						<div class="pull-<?php echo htmlspecialchars($imgfloat); ?> item-image"> <img
-						<?php if ($images->image_second_caption):
-							echo 'class="caption"'.' title="' .htmlspecialchars($images->image_second_caption) .'"';
-						endif; ?>
+						<?php if ($images->image_second_caption) : ?>
+							<?php echo 'class="caption"' . ' title="' . htmlspecialchars($images->image_second_caption) . '"'; ?>
```
Comment (chrisdavenport) [inline src/components/com_weblinks/views/category/tmpl/default_items.php]: Same here.

File: src/components/com_weblinks/views/form/view.html.php [hunk 6]
```diff
@@ -24,6 +24,13 @@ class WeblinksViewForm extends JViewLegacy
 
 	protected $state;
 
+	/**
+	 * Display the view.
+	 *
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return  mixed  A string if successful, otherwise a Error object.
```
Comment (chrisdavenport) [inline src/components/com_weblinks/views/form/view.html.php]: "a Error" should be "an Error".

File: src/components/com_weblinks/views/weblink/view.html.php [hunk 7]
```diff
@@ -20,25 +20,33 @@ class WeblinksViewWeblink extends JViewLegacy
 
 	protected $item;
 
+	/**
+	 * Display the view.
+	 *
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return  mixed  A string if successful, otherwise a Error object.
```
Comment (chrisdavenport) [inline src/components/com_weblinks/views/weblink/view.html.php]: "a Error" should be "an Error".

File: src/components/com_weblinks/views/weblink/view.html.php [hunk 8]
```diff
@@ -20,25 +20,33 @@ class WeblinksViewWeblink extends JViewLegacy
...
 		if ($item->url)
 		{
-			// redirects to url if matching id found
+			// Redirects to url if matching id found
 			JFactory::getApplication()->redirect($item->url);
 		}
 		else
 		{
-			//TODO create proper error handling
+			// TODO create proper error handling
```
Comment (chrisdavenport) [inline src/components/com_weblinks/views/weblink/view.html.php]: Should be "@TODO" ?

**Comment by zero-24** [comment]
fixed @chrisdavenport

**Pull request merged**